suite/tests/CMakeLists.txt

```diff
@@ -547,6 +548,19 @@ function(add_exe test source)
     # it hard to use other DR builds w/ these apps
     SKIP_BUILD_RPATH ON)
   if (UNIX)
+    # for common.nativeexec takeover tests, we do not support -pie/pic. newer (unix-) distributions
```

Comment:
I find "pie/pic" confusing: "pie" is clear as it means the exe itself is PIC and ET_DYN; "pic" without saying that it's applied to the exe makes me think we don't support .so files or sthg.  I would just remove the "/pic" as it does not add anything.